otter/convergence/gathering.py

```diff
@@ -138,6 +141,20 @@ def get_scaling_group_servers(tenant_id, group_id, now,
     yield do_return(servers)
 
 
+def get_all_stacks(stack_tag=None, batch_size=100):
+    query = {}
+
+    if stack_tag is not None:
+        query['tags'] = stack_tag
+
+    return list_stacks_all(query)
+
+
+def get_scaling_group_stacks(tenant_id, group_id, now,
```

Comment:
An artifact from matching the interface of `get_all_launch_server_data` and then passing arguments through. Removed now.

Also, `get_scaling_group_servers` doesn't use `now` either. I would think that the linter would catch unused method arguments...